app/Providers/EventServiceProvider.php

```diff
@@ -209,27 +208,6 @@ private function deleteFormSubmitted($event)
         }
     }
 
-    /**
-     * Populate project admin details
-     * Dispatched when a Form is submitted on the front-end, before the Submission is created.
-     *
-     * @param $event
-     */
-    private function updateProjectAdmin($event)
-    {
-        $project_id = $event->submission->get('project_id');
-        if (isValidUUID($project_id)) {
-            $projectAdmin = DB::connection('pgsql_2')
```

Comment:
There must be some config/env settings to remove, corresponding to things like `pgsql_2`, too right?

Reply:
it uses these env vars:

> 'pgsql_2' => [
>     'driver' => 'pgsql',
>     'url' => env('DATABASE_URL_2'),
>     'host' => env('DB_HOST_2', '127.0.0.1'),
>     'port' => env('DB_PORT_2', '5432'),
>     'database' => env('DB_DATABASE_2', 'forge'),
>     'username' => env('DB_USERNAME_2', 'forge'),
>     'password' => env('DB_PASSWORD_2', ''),
>     'charset' => 'utf8',
>     'prefix' => '',
>     'prefix_indexes' => true,
>     'schema' => 'public',
>     'sslmode' => 'prefer',
> ],

these can be removed @aricatalyze